CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,56 @@
+* We are kind, welcoming and courteous to everyone.
+* We are respectful of others, their positions, their skills, their commitments and their efforts.
+* We are attentive in our communications, whether in person or online, and we are tactful and respectful when approaching differing views and experiences.
+* We are aware that language shapes reality. Thus, we use inclusive, gender-neutral language in the documents we provide and when we talk to people. When referring to a group of people, we aim to use gender-neutral terms like "team", "folks" or "everyone". <sup>[1][ref-gender-neutral-docs]</sup>
+* We respect that people have differences of opinion and criticize constructively.
+
+## Unacceptable Behavior
+
+* Do not be mean or rude.
+* Do not discriminate against anyone.
+* Sexism and racism of any kind (including sexist and racist "jokes"), demeaning or insulting behaviour and harassment are seen as direct violations to this Code of Conduct. Harassment includes offensive verbal comments related to age, body size, culture, ethnicity, gender expression, gender identity, level of experience, nationality, personal ability or disability, physical appearance, physical or mental difference, race, religion, set of skills, sexual orientation, socio-economic status, and subculture. Harassment also includes sexual images in public spaces, deliberate intimidation, stalking, following, harassing photography or recording, inappropriate physical contact, and unwelcome sexual attention.
+* Respect that some individuals and cultures consider the casual use of profanity offensive and off-putting.
+* Derailing, tone arguments and otherwise playing on people's desires to be nice are not welcome, especially in discussions about violations to this Code of Conduct.
+* Please avoid unstructured critique.
+* Likewise any spamming, trolling, flaming, baiting or other attention-stealing behaviour is not welcome.
+* Publishing others' private information, such as a physical or electronic address, without explicit permission
+* Other conduct which could reasonably be considered inappropriate in a professional setting
+
+## Responsibilities
+
+The members of the core team and project maintainers are responsible for clarifying the standards of acceptable behavior and are expected to take appropriate and fair corrective action in response to any instances of unacceptable behavior.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the project core team members or owner via [email][email]. The project team will review and investigate all complaints, and will respond in a way that it deems appropriate to the circumstances. The project team is obligated to maintain confidentiality with regard to the reporter of an incident. Further details of specific enforcement policies may be posted separately.
```

Comment:
Add our communicode email addresses?

Reply:
The main idea of this (broken) link was to add a support email address like e.g. communikey-support@communicode.de, but I think we should start by adding a [`.mailmap`][git-mm]